datasets/woningbouwplannen/woningbouwplannen.json

```diff
@@ -122,8 +122,7 @@
           },
           "buurten": {
             "type": "array",
-            "relation": "bag:buurt",
-            "$comment": "relation gebieden:buurten",
+            "relation": "gebieden:buurten",
```

Comment:
Omdat het een niet-temporele relatie betreft naar een temporele tabel, moet het `relation` veldje ook de kolomnaam bevatten. Dat is nodig om te signaleren dat het om een speciaal type "losse" relatie gaat. Dus `relation` moet hier worden `gebieden:buurten:identificatie`.

Reply:
Dit moeten we nog ergens goed documenteren.

Reply:
> Dit moeten we nog ergens goed documenteren.

Heb het onder de US documenteren gezet als aandachtspunt